### PR TITLE
Burn down the house

### PR DESCRIPTION
**Pull request by umajosiah**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2366)

### CODE REVIEW

File: english/lamping/lamping.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 As you begin scrolling through your feed on IG and using your 
 super-retweeting abilities, the realities of the universe fade...
+However, you do not notice that there is a small spark of fire burning
+right beside you. As the fire increases, you begin to smell the burn 
+from deep in your sleep,
+
+Do you:
+
+wake up hurriedly and run out of the house or
+[continue to enjoy the dream you were having of your crush](burn-down-the-house/house-burn.md)
```
Comment (casey-collab) [inline english/lamping/lamping.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/burn-down-the-house/house-burn.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+As you continue sleeping and enter into a slumber, you don't realise when the entire
```
Comment (casey-collab) [inline english/burn-down-the-house/house-burn.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/burn-down-the-house/house-burn.md (hunk 2 above)
Comment (umajosiah) [inline english/burn-down-the-house/house-burn.md]: Please take another look!

**Comment by umajosiah** [comment]
Please take another look! @casey-collab

File: english/lamping/lamping.md [hunk 3]
```diff
@@ -1,2 +1,10 @@
 As you begin scrolling through your feed on IG and using your 
 super-retweeting abilities, the realities of the universe fade...
+However, you do not notice that there is a small spark of fire burning
+right beside you. As the fire increases, you begin to smell the burn 
+from deep in your sleep,
+
+Do you:
+
+wake up hurriedly and run out of the house or
+[continue to enjoy the dream you were having of your crush](english/burn-down-the-house/house-burn.md)
```
Comment (casey-collab) [inline english/lamping/lamping.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/burn-down-the-house/house-burn.md (hunk 2 above)
Comment (casey-collab) [inline english/burn-down-the-house/house-burn.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by umajosiah** [comment]
Please take another look! @casey-collab

**Comment by casey-collab** [comment]
Looks great! :star:

**Pull request merged**